packages/types/src/codec/utils/encodeTypes.ts

```diff
@@ -106,7 +106,7 @@ function encodeTuple (typeDef: Pick<TypeDef, any>): string {
   return `(${
     sub
       // eslint-disable-next-line @typescript-eslint/no-use-before-define
-      .map((type: TypeDef): string => encodeType(type))
+      .map((type: TypeDef): string => displayType(type))
```

Comment:
Is this intended? This has got an impact on more than just contracts and actually is now inconsistent with all the rest, see for instance Struct which uses the sub-encoding, which does `encodeType`.